ui/raidboss/data/triggers/o11s.js

```diff
@@ -33,6 +83,29 @@
           };
         }
       },
+      groupTTS: function(data, matches) {
+        return {
+          en: 'tankbuster',
+          de: 'basta',
+          fr: 'tankbuster',
+        };
+      },
+    },
+    {
+      id: 'O11S Atomic Ray',
+      regex: / 14:326C:Omega readies Atomic Ray,/,
+      tts: function(data) {
+        if (data.role == 'healer') {
+          return {
+            en: 'AOE',
```

Comment:
In general, aoes are not things that I think need triggers.  For small aoes like atomic ray, folks will live through regardless.  Also, cast bars generally only last one gcd, so it's not like you can do much about it if you aren't prepped or don't want to burn a swift.  This is as compared to a tank buster where yeah optimal play involves hitting your cooldowns as far ahead as possible (usually), but if you forget, you can still slam two or three cooldowns in before the tank buster hits you.